v3/src/data-interactive/resource-parser.ts

```diff
@@ -158,7 +162,29 @@ export function resolveResources(
   }
 
   if ("adornmentList" in resourceSelector && isGraphContentModel(result.component?.content)) {
-    result.adornmentList = result.component.content.adornmentsStore.adornments ?? []
+    const graphPlotType = result.component?.content.plotType
+    const adornmentList = result.component.content.adornmentsStore.adornments.reduce((list, adornment) => {
+      if (isCountAdornment(adornment)) {
+        // If the Count adornment is present, we add a separate Percent adornment item to the list. Even though
+        // Percent is part of the Count adornment, clients may not be aware of that since the UI presents them
+        // as separate entities.
+        const percentAdornment = {
+          ...adornment,
+          isVisible: adornment.showPercent,
```

Comment:
Should this be `adornment.showPercent && adornment.isVisible`? This also makes me wonder if `isVisible` is correct for the count adornment.

Reply:
Yes, you're right. Thanks. And I do think it makes sense to adjust the Count adornment's `isVisible` value, too. I've done that in the latest commit. Does that look good?